src/util/rocksdb/mod.rs

```diff
@@ -153,7 +192,9 @@ fn check_and_open(
         cfs_v.push(cf);
         match cfs_opts.iter().find(|x| x.cf == *cf) {
             Some(x) => {
-                cfs_opts_v.push(x.options.clone());
+                let mut tmp = CFOptions::new(x.cf, x.options.clone());
```

Comment:
I think L188 to the end of function can be removed now.

Reply:
Actually we don't need to add new column family or to drop deprecated column family. So we can make thing more simple here.